hooks/use-cancel-attendance.ts

```diff
@@ -0,0 +1,34 @@
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+import { usePayload } from "./use-payload";
+import { useStorage } from "./use-storage";
+
+function makeKey(eventId: number | string) {
+  return `event-attendance-${eventId}`;
+}
```

Comment:
Ah I don't like that this is duplicated. I know I did it somewhere myself, but I think in this case, you should just move the whole hook to the `use-attendance.ts` file.